fbw-a380x/src/systems/systems-host/systems/LegacyFuel.ts

```diff
@@ -19,6 +24,10 @@ import { FuelSystemEvents } from 'systems-host/systems/FuelSystemPublisher';
 export class LegacyFuel implements Instrument {
   private static NUMBER_OF_TRIGGERS = 42;
   private static NUMBER_OF_JUNCTIONS = 13;
+  private static NUMBER_OF_VALVES = 60;
+
+  //these Valves are set to true in the FLT files so we dont want to set them to false
```

Comment:
```suggestion
  /** These Valves are set to true in the FLT files so we dont want to set them to false. */
```
Better to make it a JSDoc so people will see it when they mouse over the property anywhere else.